chainio/txmgr/txmgr.go

```diff
@@ -142,93 +101,3 @@ func (m *SimpleTxManager) GetNoSendTxOpts() (*bind.TransactOpts, error) {
 	}
 	return noSendTxOpts, nil
```

Comment:
is this function `GetNoSendTxOpts` still being used?

Reply:
Yeah, in avsregistry writer and elcontracts writer

Reply:
Ok thanks for the sync yesterday, was helpful in understanding this. The part I don't understand now is how txsender can be used now with a privatekey signer. The flow in the writer clients is something like:
```
// get the noSendTxOpts that is passed to abigen functions
noSendTxOpts, err := w.txMgr.GetNoSendTxOpts()
// pass the noSendTxOpts to abigen to construct the tx
tx, err := w.someAbiGenContract.someAbiGenFunction(noSendTxOpts, args...)
// send the tx
receipt, err := w.txMgr.Send(ctx, tx)
```

Given that a txsender still takes a tx of type `*types.Transaction`, it seems like this would still need to be constructed via abigen, which requires calling `txMgr.GetNoSendTxOpts()` the same way? So it looks to me like this txsender interface is not self contained and still requires using a txMgr.

Reply:
The existing writers will still be using txMgr to interact with the chain since txsender isn't meant to replace txMgr. It just provides more primitive methods that txMgr uses to interact with the chain

Reply:
So txsender should not be used by itself then? Ever?

Reply:
Only when you need the basic broadcasting of a transaction or retrieving a transaction status.
If you want transaction monitoring/gas updates, txmgr should handle it.